pom.xml

```diff
@@ -37,7 +37,7 @@
         <spring-security.version>5.3.4.RELEASE</spring-security.version>
         <spring-security-oauth2.version>2.5.0.RELEASE</spring-security-oauth2.version>
         <spring-security-jwt.version>1.1.1.RELEASE</spring-security-jwt.version>
-        <spring-cloud.version>2.0.7.RELEASE</spring-cloud.version>
+        <hikari.version>2.4.13</hikari.version>
```

Comment:
There exists newer versions for java 8 and 11 https://github.com/brettwooldridge/HikariCP